surefire-provider/src/main/java/org/arquillian/smart/testing/surefire/provider/SmartTestingSurefireProvider.java

```diff
@@ -102,4 +107,27 @@ private File findFirstMatchingPom(File source) {
     private boolean isAnyDebugEnabled() {
         return bootParams.getMainCliOptions().contains(CommandLineOption.LOGGING_LEVEL_DEBUG) || configuration.isDebug();
     }
+
+    private Object getConsoleLogger() {
+        try {
+            Optional<Method> method = Arrays.stream(bootParams.getClass().getMethods())
+                .filter(this::isGetConsoleLoggerMethod)
+                .findFirst();
+            if (method.isPresent()) {
+                return method.get().invoke(bootParams);
+            }
+        } catch (IllegalAccessException | InvocationTargetException e) {
+            new DefaultLoggerFactory()
+                .getLogger()
+                .warn(NOT_COMPATIBLE_MESSAGE);
+        }
+        return null;
```

Comment:
The same as previous comment, if someday Surefire does not have a method called `getConsoleLogger` then a NPE will be thrown.

Reply:
the console logger is intended to be used internally only in `SurefireProviderLogger` implementation - it is not exposed anywhere